CMakeLists.txt

```diff
@@ -64,7 +68,8 @@ string(APPEND CMAKE_CXX_FLAGS " -Wbuiltin-macro-redefined")
 string(APPEND CMAKE_CXX_FLAGS " -pedantic")
 string(APPEND CMAKE_CXX_FLAGS " -Werror")
 if(IS_LEAKCHECK)
-    string(APPEND CMAKE_CXX_FLAGS " -fsanitize=leak")
+    string(APPEND CMAKE_CXX_FLAGS " -fsanitize=leak -fsanitize=address -fno-omit-frame-pointer")
```

Comment:
OOH. Thanks for this change :) Can we always run with `-fno-omit-frame-pointer`, rather than only doing it when using `IS_LEAKCHECK`?

 Omitting the frame pointer is the silliest build default ever on new hardware. The optimizer doesn't need an extra general purpose register to work with, but debugging and performance tools certainly work much better with a frame pointer.

Reply:
Yes, I think so too, the optimization when removing the frame pointer is minor. Making it mandatory is very useful for debugging and performance measuring tools.